hw5/hw5.sql

```diff
@@ -0,0 +1,137 @@
+use baseball;
+
+CREATE OR REPLACE TABLE Features as
+WITH HOME_TEAM as
```

Comment:
Good use of using cte's to increase your query time. I ran your query and it took up to 0.5s. You also avoided using self joins to calculate your statistics which increases the query time